src/ReactLeafletMap/layers/DiagramLayer/index.js

```diff
@@ -16,19 +16,30 @@ class DiagramLayer extends VectorLayer {
         return this.getDiagramLeafletStyle(feature, defaultStyleObject);
     }
 
-    getDiagramAccentedStyle(feature, defaultStyleObject, accentedStyleObject) {
-        const style = {...defaultStyleObject, ...accentedStyleObject};
+    getDiagramAccentedStyle(feature, defaultStyleObject, styleDefinition) {
```

Comment:
What's the difference between styleObject and styleDefinition?

Reply:
Better naming would be defaultStyleObject and accentedStyleObject - both objects are in panther style notation